app/steps/ui/copies/overseas/template.html

```diff
@@ -15,12 +15,11 @@
     }) %}
         {% if fields.isFeesApiToggleEnabled.value === "true" %}
             <p class="govuk-body">{{ content.paragraph1 | safe }}</p>
-            {{ unorderedList([
```

Comment:
if the bullet1 and 2 are not being used, do we still need it in the json file?

Reply:
Remove the bullets points in welsh translation :)